src/controllers/house.controllers.js

```diff
@@ -81,3 +81,19 @@ export const getHouseDetails = async (id) => {
 export const changeOwner = async (id, userId) => {
   return House.findByIdAndUpdate(id, { owner: userId }).exec();
 };
+
+export const editHouse = async (data, id) => {
+  try {
+    const house = await House.findOneAndUpdate(
+      {
+        _id: id,
+      },
+      data
```

Comment:
Some additional handling for photos should be added in the future so that only deleted ones are deleted and new ones created to avoid recreating existing ones in the database if something changes in the list of photos.